cmake/utils.cmake

```diff
@@ -28,13 +28,15 @@ endfunction()
 
 # Turn on warnings on the given target
 function(spdlog_enable_warnings target_name)

```

Comment:
Need to check if wee under msvc before creating this list

Reply:
Made some changes. You think this is good?

Reply:
why all the whitespace changes all of the sudden?

Reply:
Ahh, sorry.
The code in the spdlog_enable_warnings function uses 4 spaces as indentation, but when I added the if statements for msvc, my vscode indented with tabs.
But it seems it did it that way to due mixed usage, didnt notice, so I just did a convert tabs to spaces inside the utils file.

I should probably revert the space/tabs thing and it should probably be done as a single pull request just changing tabs to 4 spaces or the other way around depending on your preference.